#! /usr/bin/env python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('saturation_threshold', int_t, 0, 'threshold of saturation to get diff of hue', default=0)

exit(gen.generate(PACKAGE, "jsk_perception", "ImageTimeDiff"))
